src/screens/Drop.js

```diff
@@ -49,6 +49,10 @@ const DropPage = () => (
             specifying a vertical or horizontal alignment will cause it to be
             aligned in the center.
           </Description>
+          <Description>
+            The align property object should be memo-ized to maintain the same
```

Comment:
This might be a bit too narrow. I think the general concept is to not create a new object on each render but to prevent that by using a persistent object, either as a static constant outside the render path or via `useMemo` if using hooks. It might be worth looking for a React article explaining this and including a link as well.